R/across.R

```diff
@@ -148,19 +148,15 @@ across <- function(.cols = everything(), .fns = NULL, ..., .names = NULL) {
 #' @rdname across
 #' @export
 if_any <- function(.cols, .fns = NULL, ..., .names = NULL) {
-  structure(
-    across({{.cols}}, .fns = .fns, ..., .names = .names),
-    filter_combine = "or"
-  )
+  df <- across({{.cols}}, .fns = .fns, ..., .names = .names)
```

Comment:
Can you use the recommended style for `{{` please?

```suggestion
  df <- across({{ .cols }}, .fns = .fns, ..., .names = .names)
```